deepsearcher/cli.py

```diff
@@ -19,53 +19,57 @@ def main():
     init_config(config=config)
 
     parser = argparse.ArgumentParser(prog="deepsearcher", description="Deep Searcher.")
+    subparsers = parser.add_subparsers(dest="subcommand", title="subcommands")
+
     ## Arguments of query
-    parser.add_argument("--query", type=str, default="", help="query question or search topic.")
```

Comment:
For the backward compatibility and as a reminder,  can you reserve this line
`parser.add_argument("--query", type=str, default="", help="query question or search topic.")`
and the load one, and then check and raise an error after the `args = parser.parse_args()`, which remind  the users to change `--query` to `query` and `--load` to `load` if they still use this old manner.